packages/account/src/Components/poi-upload-complete/__test__/upload-complete.spec.js

```diff
@@ -0,0 +1,86 @@
+import React from "react"
+import { render, screen } from '@testing-library/react';
+import { Button } from '@deriv/components';
+import { PlatformContext } from "@deriv/shared";
+import UploadComplete from '../upload-complete'
+
+jest.mock('Components/poa-button', () => () => <div data-testid='dt_poa-button' />)
```

Comment:
Suggestion based on our testing documentation.
```suggestion
jest.mock('Components/poa-button', () => () => <div data-testid='dt_poa_button' />)
```

Reply:
Need to be updated in the respective component as well.